cocos/2d/CCCamera.h

```diff
@@ -172,6 +172,9 @@ class CC_DLL Camera :public Node
      */
     int getDepth() const { return _depth; }
     
+    float getFarPlane() const { return _farPlane; }
```

Comment:
Add any code comments? I know this function is clear enough, but we should give every public API a comment. 
Use comment to talk about the indent not the implementation.
Thanks.